package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "@schibstedspain/theme-basic",
-  "version": "5.18.0",
+  "version": "5.18.1",
```

Comment:
Firstly, versions should be upgrade after the merge. In addition, the new value should be: `5.19.0`